test/Makefile.am

```diff
@@ -0,0 +1,16 @@
+## This is a automake file, part of the GPTL package.
+# Copyright 2018, see the COPYRIGHT file for more information.
+
+# This is the automake file for GPTL source code directories.
+# Ed Hartnett, 5/18/18
+
+AM_CPPFLAGS = -I$(top_srcdir)/include 
+
+# Test programs to be built.
+#check_PROGRAMS = ???.c
```

Comment:
@jmrosinski I am having a hard time figuring out what is supposed to happen for make test. Is there a C test program that gets built somewhere?